src/prelude.rs

```diff
@@ -2,6 +2,8 @@
 
 pub use crate::flash::FlashExt as _stm32f3xx_hal_flash_FlashExt;
 pub use crate::gpio::GpioExt as _stm32f3xx_hal_gpio_GpioExt;
+pub use crate::hal::digital::v2::StatefulOutputPin as _embedded_hal_digital_StatefulOutputPin;
+pub use crate::hal::digital::v2::ToggleableOutputPin as _embedded_hal_digital_ToggleableOutputPin;
```

Comment:
These re-exports needed to be guarded by `#[cfg(feature = "unproven")]`, right? Might be nice to re-export the `OutputPin` and `InputPin` traits too.

Reply:
Yeah, you are right. I've considered it myself. Btw. is the cfg attribute always line based? Or is there another to apply it to a whole block / paragraph?

Reply:
So far as I know, it's line based only. I've used [`cfg_if`](https://github.com/alexcrichton/cfg-if) with success for block based application.